packages/frontend/components/composed/journal/FeaturedJournals/FeaturedJournal.tsx

```diff
@@ -40,6 +40,9 @@ export default function FeaturedJournal({ data, index, coverHeight }: Props) {
       >
         <Styled.ItemText as="a">
           <Styled.ItemTitle>{journal.title}</Styled.ItemTitle>
+          <Styled.ItemSubtitle className="t-copy">
+            {journal.subtitle}
```

Comment:
Same here - subtitles aren't supposed to be required, so check before adding the div